webv2/primarykey/primarykey_handler.go

```diff
@@ -100,7 +100,7 @@ func PrimaryKey(w http.ResponseWriter, r *http.Request) {
 
 	}
 
-	UpdatePrimaryKeyAndSessionFile(pkRequest)
+	UpdatePrimaryKey(pkRequest)
 
```

Comment:
we should add `session.UpdateSessionFile()` after this and test a scenario of updating PK from UI

Reply:
Added. Thanks.